open_instruct/finetune.py

```diff
@@ -363,92 +373,78 @@ def __post_init__(self):
         ):
             raise ValueError("Cannot provide two dataset selection mechanisms.")
 
+        if self.chat_template_name is None or self.chat_template_name not in CHAT_TEMPLATES:
```

Comment:
```suggestion
        if self.chat_template_name not in CHAT_TEMPLATES:
```

If we want to use the chat template from the model itself, we should add another option like `chat_template_name = "default"`

Reply:
Probably we can allow users to provide a tokenizer name here as the chat_template_name? For example, if the user provides `meta-llama/Meta-Llama-3-8B-Instruct`, we will copy its template to be ours.